lib/structs/events/common.cpp

```diff
@@ -169,6 +169,32 @@ to_json(json &obj, const VideoInfo &info)
         obj["xyz.amorgan.blurhash"] = info.blurhash;
 }
 
+void
+from_json(const json &obj, LocationInfo &info)
+{
+    if (obj.find("thumbnail_url") != obj.end())
```

Comment:
You should be able to use

```suggestion
    if (obj.contains("thumbnail_url"))
```

nowadays. Nitpick, since I haven't updated the old usages of that yet, but... if you don't mind, I think that would be cleaner.

Reply:
Sure will update that👍